application/serializers.py

```diff
@@ -146,6 +138,31 @@ def get_social_media_image_sv(self, obj):
     def get_social_media_image_en(self, obj):
         return image_serializer(obj.social_media_image_en)
 
+    class Meta:
+        model = models.BannerPages
+        fields = [
```

Comment:
Is there a reason you're not using an `exclude` list here?

Reply:
Since Banner is still a Page object and I don't want to expose any fields other than these fields in the LandingPage serializer. I think it's better to define the serialized fields explicitly here. Besides, excluding non-related banner fields would have had the same line of code any way :P